spec/spec_helper_local.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'pathname'
+dir = Pathname.new(__FILE__).parent
```

Comment:
Should it be replaced with a simple line? I think this is equivalent:
```suggestion
dir = __dir__
```